pmd-apex/src/main/resources/rulesets/apex/quickstart.xml

```diff
@@ -250,6 +250,9 @@
          <property name="cc_block_highlighting" value="false" />
       </properties>
    </rule>
+   <rule ref="category/apex/bestpractices.xml/ApexUnitTestClassShouldHaveRunAs" message="Apex unit test classes should have at least one System.runAs() call">
+      <priority>3</priority>
+   </rule>
```

Comment:
Just use a simpe reference without overriding any value. I don't know, why the messages for the other rule refs have been overridden... it is unnecessary, unless you really want a different message...
Also the priority is inherited from the original rule definition.

```suggestion
   <rule ref="category/apex/bestpractices.xml/ApexUnitTestClassShouldHaveRunAs" />
```

Actually the messages _are_ different...

Reply:
> 

You're right, initially I did not call the constructor in the java class to define the message and priority.
Thanks !